File: auto-docs/executables/python/bar_marker_array.py
import plotly.plotly as py
from plotly.graph_objs import *
py.sign_in('TestBot', 'r1neazxo9w')

data = Data([
    Bar(
        x=[1, 2, 3, 4],
        y=[5, 4, -3, 2],
        marker=Marker(
            color=['#447adb', '#447adb', '#db5a44', '#447adb']
        )
    )
])
plot_url = py.plot(data, filename='bar-marker-array', auto_open=False)

File: auto-docs/executables/julia/bar_marker_array.jl
using Plotly
Plotly.signin("TestBot", "r1neazxo9w")

data = [
  [
    "x" => [1, 2, 3, 4], 
    "y" => [5, 4, -3, 2], 
    "marker" => ["color" => ["#447adb", "#447adb", "#db5a44", "#447adb"]], 
    "type" => "bar"
  ]
]
response = Plotly.plot(data, ["filename" => "bar-marker-array", "fileopt" => "overwrite"])
plot_url = response["url"]